soc/nxp/mcx/soc.yml

```diff
@@ -30,6 +30,20 @@ runners:
       - qualifiers:
         - mcxn947/cpu0
         - mcxn947/cpu1
+      - qualifiers:
```

Comment:
I guess this is needed for supporting MCUBoot, since you want to only erase once? I'm confused why this wouldn't be the default when running MCUBoot sysbuild images

Reply:
> I'm confused why this wouldn't be the default when running MCUBoot sysbuild images

Me too. 
Logically, it should be the default behavior.
Better to ask Nordic - authors of this implementation.

Reply:
@nordicjm any idea why this isn't the default?

Reply:
It requires configuring because the option might be for one board target or multiple, the flash running is outside of the build system and just does comparisons on strings